packages/vaadin-grid/src/vaadin-grid.js

```diff
@@ -385,18 +385,40 @@ class GridElement extends ElementMixin(
       },
 
       /**
+       * **Deprecated**
+       *
+       * See [`allRowsVisible`](#/elements/vaadin-grid#property-allRowsVisible)
```

Comment:
I think we would want to use [JSDoc links](https://jsdoc.app/tags-inline-link.html) here? Was there an existing example of this style of link somewhere in the existing code?

Reply:
I used JSDocs' links first but couldn't make it work. It doesn't generate an anchor tag in the docs (`yarn docs`).
Yeah this the format that is used to make a link throughout the project, like in the header section of this file.
I can't find any JSDoc link.